File: cf-1919a.py
T = int(input())
while T:
	turns = input()
	turns = turns.split()
	A = int(turns[0])
	B = int(turns[1])
	if A == B:
		print("Bob")
	elif A % 2 == 0 and B % 2 == 0:
		print("Bob")
	elif A % 2 != 0 and B % 2 != 0:
		print("Bob")
	elif A % 2 != 0 and B % 2 == 0:
		print("Alice")
	else:
		print("Alice")
	T -= 1

File: cf-1919a.cpp
#include <iostream>
#define ll long long
using namespace std;

int main () {
    int t;
    ll a, b;
    
    cin >> t;
    while (t--) {
        cin >> a >> b;
        if (a == b)
            cout << "Bob\n";
        else if (a % 2 != 0 && b % 2 == 0)
            cout << "Alice\n";
        else if (a % 2 == 0 && b % 2 != 0)
            cout << "Alice\n";
        else if (a % 2 == 0 && b % 2 == 0)
            cout << "Bob\n";
        else
            cout << "Bob\n";
    }

    return 0;
}
